browse/static/js/coreRecommender.js

```diff
@@ -0,0 +1,62 @@
+function getCORERecCookie() {
+    var coreCookie = "arxiv_core";
+    var dc = document.cookie;
+    var prefix = coreCookie + "=";
+    var begin = dc.indexOf(prefix);
+    var end = document.cookie.indexOf(";", begin);
+    var returnVal;
+    if (end == -1) {
+        end = dc.length
+    }
+    returnVal = decodeURIComponent(dc.substring(begin + prefix.length, end));
+    return returnVal
+}
+function setCORERecCookie() {
+    var coreCookie = "arxiv_core";
+    var value = "disabled";
+    var cv = getCORERecCookie();
+    console.log("cv");
+    console.log(cv);
```

Comment:
Two lines of `console.log` are called separately and may be spited by a network response running concurrently by another script. It's harder to read logs so but `console.log` supports any count of arguments.

```suggestion
    console.log("cv", cv);
```